spec/unit/puppet_x/sql_connection_spec.rb

```diff
@@ -8,8 +8,11 @@
 
   def stub_connection
     @connection = mock()
+    error = mock()
```

Comment:
Now is not the time, but should really move away from using `mocha` for stubs and just using core rspec doubles.

Reply:
It took me a while to figure out why all of my reading on how to do mocking in rspec was coming up with a different syntax.